data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we need to get some sign of your good intentions. So if you are really on our side you should send us some gold and a couple of other wares on a regular basis.]])..
+         _([[At most we are interested in any wares that can be used to forge weapons. I mean real weapons like our axes, and not the sort of toothpicks you use. So feel free to send us 25 logs, 16 coal, 9 iron and 4 gold.]])
+         .. paragraphdivider() ..
+         _([[Oh I almost forgot this one. We demand to have your princess in our headquarters as well to coordinate our warplannings.]]))
+   },
+   {
+      title = _"Pitfall!",
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm you, Jundlina! This behaviour is truely babaric! I fear this is a trap to keep you as a hostage. And the demand itself is really an insult. I strongly recommend to decline this unsubstantiated claim.]]))
+   },
+   {
+      title = _"Jundlina Concedes",
+      body = jundlina(_"Jundlina",
+         -- TRANSLATORS: Jundlina
+         _([[Sidolus, I agree with you. However I think we have no choice so we need to obey to this demand. It is your obligation now to run our small people until I get back.]])
+         .. paragraphdivider() ..
+         _([[First you need to upgrade our scout’s house to a trading post. Then you need to send the demanded tributes of gold, iron, coal and logs.]]))
+         .. new_objectives(obj_trading_post)
+   }
+}
+
+trading = {
+   {
+      title = _"Trading post built",
+      body = loftomor(
+         -- TRANSLATORS: loftomor
+         _([[Sidolus, may Satul warm you! We have finished the new trading post. Hopefully this will get us back Jundlina soon]]))
+   },
+   {
+      title = _"Sidolus Agrees",
+      body = sidolus(
+         -- TRANSLATORS: sidolus
+         _([[Well done, Loftomor, well done. May Satul warm our princess and our whole people. Now we need to begin sending goods to this barbarian.]])
+         .. paragraphdivider() ..
+         _([[I think we should start with the wood deliveries as we could afford them the easiest. But we need to find more metals.]]))
+         .. new_objectives(obj_tribute)
+   }
+}
+
+tribute_not_started = {
+   {
+      title = _"Urgent Tribute Demand",
+      body = kalitath(
+         -- TRANSLATORS: Kalitath
+         _([[Strangers, we haven't received any help yet. So if you are really on our side you should start sending some wares urgently.]])..
+         _([[As we are facing great losses due to your lazyness, you should now send us double the amount of each ware as before.]])
+         .. paragraphdivider() ..
+         _([[I am sure you won't loose our alliance, wouldn't you?]]))
+   },
+   {
+      title = _"Blackmail!",
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm us all. Our so-called ally is blackmailing us and we can't punish his insult as he has our princess at his hands.]])
```

Comment:
```suggestion
         _([[May Satul warm us all. Our so-called ally is blackmailing us, and we can’t punish him for his insult as he has our princess at his hands.]])
```
I'm not sure blackmailing is the right word here, but I can't think of a better fit just now